src/components/app/app.js

```diff
@@ -12,19 +12,15 @@ import { useStyles } from './app.styles';
 import { getFromLocalStorage } from '../../services/local-storage.service';
 import { setThemeMode } from '../../redux/theme/theme.actions';
 import { getCategories } from '../../redux/categories/categories.actions';
-import { Loader } from '../loader/loader';
 import { preserveUser } from '../../redux/user/user.actions';
 import { setCountPerPage } from '../../redux/products/products.actions';
 import { getContacts } from '../../redux/contacts/contacts.actions';
 
 const App = () => {
-  const { isLoading, lightMode, location } = useSelector(
-    ({ Categories, Theme, router }) => ({
-      isLoading: Categories.loading,
-      lightMode: Theme.lightMode,
-      location: router.location.pathname
-    })
-  );
+  const { lightMode, location } = useSelector(({ Theme, router }) => ({
```

Comment:
Please move this selector to additional file.

Reply:
fixed